packages/perseus/src/widgets/__tests__/radio/choice_test.jsx

```diff
@@ -0,0 +1,201 @@
+// @flow
+
+import {render, screen} from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import * as React from "react";
+
+import "@testing-library/jest-dom"; // Imports custom mathers
+
+import Choice from "../../radio/choice.jsx";
+
+function renderChoice(options) {
+    const defaultOptions = {
+        checked: false,
+        rationale: "This is a good rationale",
+        content: "This is a possible choice",
+        correct: true,
+        disabled: false,
+        pos: 0,
+        reviewMode: false,
+        showRationale: false,
+        showCorrectness: false,
+        multipleSelect: false,
+        crossedOut: false,
+        previouslyAnswered: false,
+        ref: {current: null},
+    };
+
+    const overwrittenOptions = {...defaultOptions, ...options};
+
+    return render(<Choice {...overwrittenOptions} />);
+}
+
+describe("choice", () => {
```

Comment:
**question:** what kind of coverage is this test suite providing for choice.jsx?